kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/ConsumerSeekUtilIntegrationTest.java

```diff
@@ -195,11 +189,8 @@ void testSeekUsingExistingTokenConsumerStartsAtSpecificPositionAndCanContinueRea
         //  their offsets will increase given the published number of `recordsPerPartitions`
         int numberOfRecordsToConsume = 26;
 
-        Consumer<?, ?> testConsumer = consumerFactory.createConsumer(DEFAULT_GROUP_ID);
-        testConsumer.subscribe(
-                Collections.singletonList(topic),
-                new TrackingTokenConsumerRebalanceListener<>(testConsumer, () -> testToken)
-        );
+        Consumer<?, ?> testConsumer = consumerFactory.createConsumer(null);
```

Comment:
I don't think so. It should not really be used with a `groupid` anyway. There doesn't seem to be a nice way to check this trough. But if it is used with `groupid` and multiple instances use the same `groupid` it will throw some error probably.